### PR TITLE
Speaker pics, first pass

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/_ext/core.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def media_photo(_file, _type):
         return '/_static/img/speakers/missing.jpg'
 
     def speaker_photo(file):
-        return media_photo(file, 'speaker')
```
Comment (plaindocs) [inline docs/_ext/core.py:126]: That took me a surprisingly long time to track down.

**Comment by plaindocs** [comment]
@ericholscher didn't realize you'd already fixed up the announcement. :+1:

File: docs/_ext/core.py (hunk 1 above)
Comment (ericholscher) [reply docs/_ext/core.py:126]: Huh, I thought I had speaker pics rendering properly with previous speakers. Weird.

Comment (plaindocs) [reply docs/_ext/core.py:126]: Maybe we reverted somehow?

File: docs/conf/portland/2019/news/announcing-speakers.rst [hunk 2]
```diff
@@ -1,127 +1,77 @@
...
+`ElasticSearch <https://www.elastic.co/>`_
 
-`Elastic <https://www.elastic.co>`_
+`Knowledgeowl <https://www.knowledgeowl.com/>`_
 
-`KnowledgeOwl <https://www.knowledgeowl.com/home>`_
+`Netlify <https://www.netlify.com/wtd>`_
 
-`Cloud Native Computing Foundation <https://www.cncf.io>`_
+`CNCF <https://www.cncf.io/>`_
```
Comment (ericholscher) [inline docs/conf/portland/2019/news/announcing-speakers.rst]: I think the full name is better.

File: docs/conf/portland/2019/news/announcing-speakers.rst [hunk 3]
```diff
@@ -1,127 +1,77 @@
...
 
-`StickerMule <https://www.stickermule.com/supports/writethedocs-portland>`_
+`StickerMule <https://www.stickermule.com/supports/opensource>`_
```
Comment (ericholscher) [inline docs/conf/portland/2019/news/announcing-speakers.rst]: Old URL here is correct.

File: docs/conf/portland/2019/news/announcing-speakers.rst [hunk 4]
```diff
@@ -1,127 +1,77 @@
...
 
-Want to lend a hand? Head over to our `sponsorship prospectus <http://www.writethedocs.org/conf/portland/2019/sponsors/prospectus/>`_ for
-details on how your company can get involved! We have several types of
-sponsorship available, including the option to participate in our job  
-fair that we'll be hosting on Tuesday.
+Want to lend a hand? Head over to our `sponsorship prospectus <http://www.writethedocs.org/conf/portland/2019/sponsors/prospectus/>`_ for details on how your company can get involved! We have several types of sponsorship available, including tabes at the job fair that we'll be hosting on Tuesday.
```
Comment (ericholscher) [inline docs/conf/portland/2019/news/announcing-speakers.rst]:
```suggestion
Want to lend a hand? Head over to our `sponsorship prospectus <http://www.writethedocs.org/conf/portland/2019/sponsors/prospectus/>`_ for details on how your company can get involved! We have several types of sponsorship available, including tables at the job fair that we'll be hosting on Tuesday.
```

File: docs/conf/portland/2019/news/announcing-speakers.rst [hunk 5]
```diff
@@ -1,127 +1,77 @@
...
 
-We've got a couple of excellent volunteers – Mo Nishiyama and Alicia
-Duncan – who are heading up our Explore Portland committee. They're
-already hard at work putting together resources to help you plan your
-visit! Check out the `Visit Portland <http://www.writethedocs.org/conf/portland/2019/visiting/>`_ page
-on the conference website, or pop into the
-`#wtd-conferences <https://writethedocs.slack.com/messages/wtd-conferences>`_ channel
-on Slack to ask a question.
+We've got a couple of excellent volunteers – Mo Nishiyama and Alicia Duncan – who are heading up our Explore Portland committee. They're already hard at work putting together resources to help you plan your visit! Check out the `Visit Portland <http://www.writethedocs.org/conf/portland/2018/visiting/>`_ page on the conference website, or pop into the `#wtd-conferences <https://writethedocs.slack.com/messages/wtd-conferences>`_ channel on Slack to ask a question.
```
Comment (ericholscher) [inline docs/conf/portland/2019/news/announcing-speakers.rst]: Old URL for visiting page

```suggestion
We've got a couple of excellent volunteers – Mo Nishiyama and Alicia Duncan – who are heading up our Explore Portland committee. They're already hard at work putting together resources to help you plan your visit! Check out the `Visit Portland <http://www.writethedocs.org/conf/portland/2019/visiting/>`_ page on the conference website, or pop into the `#wtd-conferences <https://writethedocs.slack.com/messages/wtd-conferences>`_ channel on Slack to ask a question.
```

File: docs/conf/portland/2019/news/announcing-speakers.rst [hunk 6]
```diff
@@ -1,127 +1,77 @@
...
-without your support and proposals. **Please consider submitting your
-talks to Write the Docs meetup near you, as they are always looking for
-more speakers!**
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure. We're really excited about the presentations we've got this year – we hope you are too!
+
+.. datatemplate::
+   :source: /_data/{{year}}.{{shortcode}}.speakers.yaml
+   :template: {{year}}/speakers-simple-list.rst
+   :include_context:
+
```
Comment (ericholscher) [inline docs/conf/portland/2019/news/announcing-speakers.rst:24]: Better solution 👍

**Review by ericholscher — APPROVED** [review]

**Comment by plaindocs** [comment]
Thanks for the :eyes:  @ericholscher !
@bethaitman this needs merging before you mail the newsletter :-p

**Pull request merged**